modules/raft-api/src/main/java/org/apache/ignite/internal/raft/configuration/LogStorageConfigurationSchema.java

```diff
@@ -26,18 +26,42 @@
 /** Configuration of the Raft log storage. */
 @Config
 public class LogStorageConfigurationSchema {
+    public static final int DEFAULT_MAX_CHECKPOINT_QUEUE_SIZE = 10;
+
+    public static final int DEFAULT_SEGMENT_FILE_SIZE_BYTES = Integer.MAX_VALUE;
+
+    public static final int UNSPECIFIED_MAX_LOG_ENTRY_SIZE = -1;
+
     /**
      * Maximum size of the log storage checkpoint queue.
      */
     @Value(hasDefault = true)
     @Range(min = 1)
-    public int maxCheckpointQueueSize = 10;
+    public int maxCheckpointQueueSize = DEFAULT_MAX_CHECKPOINT_QUEUE_SIZE;
 
     /**
      * Size of a segment file in bytes.
      */
-    @SuppressWarnings("PointlessArithmeticExpression") // Suppressed for better readability.
     @Value(hasDefault = true)
-    @Range(min = 1 * KiB, max = Integer.MAX_VALUE)
-    public long segmentFileSizeBytes = Integer.MAX_VALUE;
+    @Range(min = 4 * KiB, max = Integer.MAX_VALUE)
```

Comment:
Either an explicit max value is unnecessary here, or an explicit max value should be specified for the `maxCheckpointQueueSize`, for consistency.